src/test/resources/expected/imp.kore.golden

```diff
@@ -1,3228 +1 @@
-                                                            , variableSort =
-                                                                SortActualSortSortActual
-                                                                        { sortActualName = Id Object "AExp"
-                                                                        , sortActualSorts = []
-                                                                        }
-                                                            })
-                                                        ]
-                                                    })
-                                                ]
-                                            })
-                                    })
-                            , impliesSecond =
-                                MetaPattern (ApplicationPattern Application
-                                    { applicationSymbolOrAlias =
-                                        SymbolOrAlias
-                                            { symbolOrAliasConstructor = Id Meta "#provable"
-                                            , symbolOrAliasParams =
-                                                [ SortVariableSort (SortVariable (Id Meta "#R"))
-                                                ]
-                                            }
-                                    , applicationChildren =
-                                        [ ObjectPattern (EqualsPattern Equals
-                                            { equalsOperandSort =
-                                                SortActualSortSortActual
-                                                        { sortActualName = Id Object "AExp"
-                                                        , sortActualSorts = []
-                                                        }
-                                            , equalsResultSort =
-                                                SortVariableSort (SortVariable (Id Object "R"))
-                                            , equalsFirst =
-                                                ObjectPattern (ApplicationPattern Application
-                                                    { applicationSymbolOrAlias =
-                                                        SymbolOrAlias
-                                                            { symbolOrAliasConstructor = Id Object "divAExp"
-                                                            , symbolOrAliasParams = []
-                                                            }
-                                                    , applicationChildren =
-                                                        [ ObjectPattern (ApplicationPattern Application
-                                                            { applicationSymbolOrAlias =
-                                                                SymbolOrAlias
-              
```

Comment:
I'm not sure this test was supposed to fail... maybe the original .kore file needs to be changed? @hreada